src/diffpy/labpdfproc/tools.py

```diff
@@ -281,7 +326,8 @@ def preprocessing_args(args):
     args.output_directory = set_output_directory(args)
     args = set_wavelength(args)
     args = set_xtype(args)
-    args = set_mud(args)
+    args = set_mud_from_zscan_file(args)
```

Comment:
Here I am thinking that during the pre-processing we first check for z-scan files and read muD from there. If there's no file /muD provided, we then proceed to ask for other information.